docs/cow-protocol/tutorials/solvers/onboard.md

```
@@ -123,7 +123,7 @@ Every week on Tuesday your solver will receive payments and rewards for settling
 
 We advise using a single rewards address that is available on all networks.
 
-[You can find more information about how we calculate and distribute rewards here.](https://docs.cow.finance/cow-protocol/reference/core/auctions/rewards). The accounting process is documented in detail [here](https://docs.cow.finance/cow-protocol/reference/core/auctions/accounting).
+[You can find more information about how we calculate and distribute rewards here.](https://docs.cow.fi/cow-protocol/reference/core/auctions/rewards). The accounting process is documented in detail [here](https://docs.cow.fi/cow-protocol/reference/core/auctions/accounting).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)